sycl/doc/KernelParameterPassing.md

```diff
@@ -90,7 +90,7 @@ The compiler-generated code for a call to such a lambda function would look like
 When offloading the kernel to a device, the lambda/function object's 
 function operator cannot be directly called with a capture object address.
 Instead, the code generated for the device is in the form of a
-�kernel caller� and a �kernel callee�.
+"kernel caller" and a "kernel callee".
```

Comment:
It turned out that it's not the only problem.
https://github.com/intel/llvm/runs/847989560

Could you try to build the documentation locally to verify the fix?